app/policies/lab_test_policy.rb

```diff
@@ -9,6 +9,10 @@ def show?
     user.full_access_roles_can? || record.user == user
   end
 
+  def new?
+    create?
```

Comment:
You don't need this as it is set by default in [application_policy.rb](https://github.com/Code-the-Dream-School/health-keeper/blob/main/app/policies/application_policy.rb#L24) that `lab_test_policy.rb` is inherited from.